enterprise/app/sidebar/sidebar.css

```diff
@@ -9,17 +9,15 @@
   overflow-x: hidden;
   font-size: 18px;
   user-select: none;
+  white-space: nowrap;
+  transition: 300ms ease-in-out;
```

Comment:
I messed around with this a bit - I'm not sure I notice a huge difference between the two since it's only 300ms.

I think I'll leave as-is for now since I feel like it might be a little nicer for toggling the sidebar back and forth to have the curves match in either direction.

I totally understand what you're saying though with the eagerness and am down to make the switch if it becomes annoying.